helm/tenant/templates/tenant.yaml

```diff
@@ -47,129 +47,129 @@ spec:
             requests:
               storage: {{ dig "size" "10Gi" . }}
       {{- with (dig "annotations" (dict) .) }}
-      annotations: {{- toYaml . | nindent 8 }}
+      annotations: {{- toYaml .tenant | nindent 8 }}
```

Comment:
how about here

Reply:
Changed it, as far as I can see, everywhere now.

Reply:
That isn't possible from my testing. When L49 - L74 switches to place .tenant after (dict)/(list) it fails with Error: `INSTALLATION FAILED: template: tenant/templates/tenant.yaml:49:16: executing "tenant/templates/tenant.yaml" at <dig "annotations" (dict) .tenant>: error calling dig: interface conversion: interface {} is nil, not map[string]interface {}`

So these lines are on purpose as is.

Reply:
I see now that `range` covers the (dict)/(list)  L49 - L74. Changing it.